R/utils-pipe.R

```diff
@@ -0,0 +1,14 @@
+#' Pipe operator
+#'
+#' See \code{magrittr::\link[magrittr:pipe]{\%>\%}} for details.
+#'
+#' @name %>%
```

Comment:
What version of R are you planning to support for this package? You could make it easier for yourself and start from `R >= 4.1.0`

Reply:
I hadn't given this much consideration and added the pipe mostly so existing code would work without much modification. 

I think R4.1.0 is probably reasonable.